### PR TITLE
DBZ-578 Process timestamps without timezones as in server timezone

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: docs/connectors/mysql.asciidoc [hunk 1]
```diff
@@ -958,6 +958,12 @@ Other than MySQL's `TIMESTAMP` data type, the other MySQL temporal types depend
 As of Debezium 0.7 `adaptive_time_microseconds` mode was introduced and is the default `time.precision.mode` for the MySQL connector. Mode `adaptive` was marked as deprecated.
 ====
 
+[NOTE]
+====
+As of Debezium 0.8 all time related types that does not have associated timezone are related to the timezone in which the database is operating.
```
Comment (gunnarmorling) [inline docs/connectors/mysql.asciidoc]: "Are related" sounds a bit vague. How about giving an example, clarifying that such columns are converted to UTC in the CDC messages?

File: docs/connectors/postgresql.asciidoc [hunk 2]
```diff
@@ -1206,6 +1212,12 @@ The following configuration properties are _required_ unless a default value is
 When `true` the delete operations are represented by a delete event and a subsquent tombstone event. When `false` only a delete event is sent. +
 Emitting the tombstone event (the default behavior) allows Kafka to completely delete all events pertaining to the given key once the source record got deleted.
 
+|`server.zone.offset` +
+0.8.0 and later
+|
+| Specifies timezone offset of the timezone where database server is located.
+This value is used to normalize timestamps without timezones to the value as defined by server timezone.
```
Comment (gunnarmorling) [inline docs/connectors/postgresql.asciidoc]: "normalize" should also mention the fact that UTC is the target format.

**Comment by jpechane** [comment]
@gunnarmorling Reworded

**Comment by gunnarmorling** [comment]
Merged into 0.8 with a few adjustments. Thanks, @jpechane!

**Pull request closed without merging**